VMware.CloudFoundation.PowerManagement.psd1

```diff
@@ -52,15 +52,19 @@
     # Modules that must be imported into the global environment prior to importing this module
     RequiredModules   = @(
         @{
-            ModuleName    = 'Posh-SSH';
+            ModuleName    = 'VMware.PowerCLI'
+            ModuleVersion = '12.5.0'
+        }
+        @{
+            ModuleName    = 'Posh-SSH'
             ModuleVersion = '3.0.8'
         }
         @{
-            ModuleName    = 'PowerVCF';
+            ModuleName    = 'PowerVCF'
             ModuleVersion = '2.4.0'
```

Comment:
Should we bunp to v2.4.1 like the other modules?

Reply:
Yes - I've updated it in the upcoming pull request. This one address only formation and some immediate issues.